docs/styleguide/PyGuide.md

```diff
@@ -102,39 +101,19 @@ arr2 = [
 
 ## 3 Python Language Rules
 
-<a id="s3.1-pylint"></a>
-<a id="31-pylint"></a>
-<a id="pylint"></a>
-
-### 3.1 PyLint
```

Comment:
Please add description about ruff instead of pylint, with links and useful tags.

Reply:
Done